packages/tools/devtools/devtools-browser-extension/e2e-tests/tsconfig.json

```
@@ -0,0 +1,14 @@
+{
+	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"compilerOptions": {
+		"target": "es2018",
+		"module": "esnext",
+		"outDir": "dist/tsc",
```

```suggestion
		"outDir": "dist/e2e-tests",
```
Since this is technically a separate TypeScript project from the main source code (which writes its transpiled files to `dist/tsc`), we'll want the files to be written to a different location under `dist/`. This package is not published so it doesn't matter as much as in a package where the contents of the `dist/` folder are important, but we should still split them to keep things clean and manageable.